Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml

```diff
@@ -5,20 +5,20 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/content_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@drawable/contact_diary_onboarding_background">
```

Comment:
Why is this removed? Background should be greyish in dark mode in the onboarding screen (not only toolbar)

Reply:
I put it back although the different style does not make sense to